src/Core/Models/Table/TaxRate.cs

```diff
@@ -0,0 +1,18 @@
+namespace Bit.Core.Models.Table
+{
+   public class TaxRate: ITableObject<string>
+   {
+      public string Id { get; set; }
+      public string Country { get; set; }
+      public string State { get; set; }
+      public string PostalCode { get; set; }
+      public decimal Rate { get; set; }
+      public bool Active { get; set; } 
+
+      public void SetNewId()
+      {
+         // Id is created by Stripe, should exist before this gets called
```

Comment:
Should we have a different, external id? I'm not sure tying our primary key to an external source is the best thing.

Reply:
I ended up of the mind that having a separate PK was redundant and useless. We do have a 1-1 relationship going between our TaxRate data and Stripe's TaxRate data, and at the end of the day it is going to be Stripe's ID keeping everything unique even if we add another ID.

I had an auto-incrementing int in here at first, but it was never used for anything, so I took it out. I could see a to have a BW-owned Id if there is a chance we would:

1.) Stop using Stripe.
2.) Whatever the new payment solution was didn't require a large scale refactor of the TaxRate logic anyway.

(2) in particular doesn't seem likely with how much control Stripe wants over the whole thing.

Regardless, there's no real harm in having a different PK, so let me know if you still want to go that route @kspearrin

Reply:
I don't mind having it if you think that's the best way. Thought I would just bring up the question.

Reply:
If we have to refactor later we just need to remember this, it's more efficient now, but if we change providers it becomes technical debt as it breaks the current "provider" model